tests/Issues/issue450Test.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Issues;
+
+use Mockery;
+use Mpdf\Color\ColorConverter;
+use Mpdf\CssManager;
+use Mpdf\Image\ImageProcessor;
+use Mpdf\Image\Svg;
+use Mpdf\Language\LanguageToFont;
+use Mpdf\Language\ScriptToLanguage;
+use Mpdf\Otl;
+use Mpdf\SizeConverter;
+
+/**
+ * Class Issue450Test
+ * @author Antonio Norman - softcodex.ch
+ */
+class Issue450Test extends \Mpdf\BaseMpdfTest
+{
+
+	/**
+	 * SVGs with global CSS styles, had the styles ignored
+	 */
+	public function testSvgWithGlobalStyles()
+	{
+		// Copied from  Mpdf\Image\MetricsGeneratorTest
```

Comment:
How is this comment relevant?

Reply:
As I'm not familiar with the mpdf code base, if there's something wrong at least you know where I got it from 🤨